README.rst

```diff
@@ -555,7 +556,7 @@ Choose from the list of available rules:
    | *Rule is: risky.*
 
 * **simplified_null_return**
-   | A return statement wishing to return nothing should be simply "return".
+   | A return statement wishing to return "void" should not return "null".
```

Comment:
to prove that it does not apply willingly
utests in this repo wont test custom fixers which have a definition.


I just don't want to have default parameters at DTO objects...